src/webgl/p5.Geometry.js

```diff
@@ -1520,6 +1522,7 @@ class Geometry {
       }
     }
     this.lineVertices.push(...a, ...b, ...a, ...b, ...b, ...a);
+    if(!this.renderer._simpleLines){
```

Comment:
Similarly here, we might need to check for the existence of the renderer first:
```suggestion
    if(!this.renderer?._simpleLines){
```